index.bs

```diff
@@ -92,19 +92,15 @@ extensible.
   togglePipButton.hidden = !document.pictureInPictureEnabled ||
     video.disablePictureInPicture;
 
-  togglePipButton.addEventListener('click', function() {
+  togglePipButton.addEventListener("click", async () => {
     // If there is no element in Picture-in-Picture yet, let's request
     // Picture-in-Picture for the video, otherwise leave it.
-    if (!document.pictureInPictureElement) {
-      video.requestPictureInPicture()
-      .catch(error => {
-        // Video failed to enter Picture-in-Picture mode.
-      });
-    } else {
-      document.exitPictureInPicture()
-      .catch(error => {
-        // Video failed to leave Picture-in-Picture mode.
-      });
+    try {
+      !document.pictureInPictureElement
```

Comment:
I would recommend only using ternary expressions where the result is assigned to a variable, not purely for control flow